pkg/client/informers/externalversions/factory.go

```diff
@@ -48,11 +48,6 @@ type sharedInformerFactory struct {
 	// startedInformers is used for tracking which informers have been started.
 	// This allows Start() to be called multiple times safely.
 	startedInformers map[reflect.Type]bool
-	// wg tracks how many goroutines were started.
-	wg sync.WaitGroup
```

Comment:
Why were changes in the `factory` made? I don't follow - especially around the shutdown logic.